backend/FwLite/FwDataMiniLcmBridge/Api/UpdateProxy/UpdateMorphTypeDataProxy.cs

```diff
@@ -0,0 +1,53 @@
+using MiniLcm.Models;
+using SIL.LCModel;
+
+namespace FwDataMiniLcmBridge.Api.UpdateProxy;
+
+public class UpdateMorphTypeDataProxy : MorphTypeData
+{
+    private readonly IMoMorphType _lcmMorphType;
+    private readonly FwDataMiniLcmApi _lexboxLcmApi;
+
+    public UpdateMorphTypeDataProxy(IMoMorphType lcmMorphType, FwDataMiniLcmApi lexboxLcmApi)
+    {
+        _lcmMorphType = lcmMorphType;
+        Id = lcmMorphType.Guid;
+        _lexboxLcmApi = lexboxLcmApi;
+    }
+
+    public override MultiString Name
+    {
+        get => new UpdateMultiStringProxy(_lcmMorphType.Name, _lexboxLcmApi);
+        set => throw new NotImplementedException();
+    }
+
+    public override MultiString Abbreviation
+    {
+        get => new UpdateMultiStringProxy(_lcmMorphType.Abbreviation, _lexboxLcmApi);
+        set => throw new NotImplementedException();
+    }
+
+    public override RichMultiString Description
+    {
+        get => new UpdateRichMultiStringProxy(_lcmMorphType.Description, _lexboxLcmApi);
+        set => throw new NotImplementedException();
+    }
+
+    public override string LeadingToken
+    {
+        get => _lcmMorphType.Prefix;
+        set => throw new NotImplementedException();
```

Comment:
this and the next 3 properties should set their respective values

Reply:
Done in commit 6b537efdd.